docs/configuration.md

```diff
@@ -0,0 +1,210 @@
+# Configuration
+
+## Project Wide
+
+You can turn on specific rules by toggling them in a
+`.template-lintrc.js` file at the base of your project, or at a custom relative
+path which may be identified using the CLI:
+
+```javascript
+module.exports = {
+  extends: 'recommended',
+
+  rules: {
+    'no-bare-strings': true,
+  },
+};
+```
+
+This extends from the builtin recommended configuration ([lib/config/recommended.js](lib/config/recommended.js)),
+and also enables the `no-bare-strings` rule (see [here](docs/rule/no-bare-strings.md)).
+
+Using this mechanism allows you to extend from the builtin, and modify specific rules as needed.
+
+Some rules also allow setting additional configuration, for example if you would like to configure
+some "bare strings" that are allowed you might have:
+
+```javascript
+module.exports = {
+  rules: {
+    'no-bare-strings': ['ZOMG THIS IS ALLOWED!!!!'],
+  },
+};
+```
+
+## Configuration Properties
+
+The following properties are allowed in the root of the `.template-lintrc.js` configuration file:
+
+<table>
```

Comment:
We shouldn't be using an HTML table for this, can use markdown table instead.

Reply:
HTML was much easier to work with. I'm not sure I would say shouldn't, but it's fine to convert it. Done.